util/make_timeline.py

```diff
@@ -243,7 +254,7 @@ def main(args):
             pass
 
         # Set the time, possibly adjusting to specified phase
-        if entry["ability_id"] not in phases:
+        if entry["line_type"] == "34" or entry["ability_id"] not in phases:
```

Comment:
Probably this should be a check for not 21,22 rather than is 34, for if/when we add 03 lines.